config_src/coupled_driver/ocean_model_MOM.F90

```diff
@@ -267,6 +265,10 @@ subroutine ocean_model_init(Ocean_sfc, OS, Time_init, Time_in, wind_stagger, gas
   endif
   allocate(OS)
 
```

Comment:
Not sure what is the intent of commented code. Perhaps it is needed for ...? It would help to have a single comment to state the purpose and why it is commented (right now).

Reply:
This is a good question. I think that @MJHarrison-GFDL would be the right person to answer the question of why there are now commented out statements allocating the OS%fluxes, OS%forces and OS%flux_tmp elements of the ocean_state_type.   These are actual types in the declaration, so they can not be allocated, but at one point last autumn there was the idea to make these into pointers, but I believe that we decided to backtrack on that to avoid introducing inconsistencies with the other drivers. I don't recall more of the details about this conversation than this.  These commented-out lines could be deleted, but as they do not impact the code directly, perhaps this could be done in a subsequent commit.

Reply:
@sanAkel Thanks for pointing out this remnant comment related to a redacted version of the code mentioned by @Hallberg-NOAA  which was an attempt to handle the rotation of the fluxes and forcing types as part of the tc regressions. These comments can be safely deleted.